README.md

```diff
@@ -20,11 +20,11 @@
 
 This monorepo contains code for open-source Code PushUp NPM packages:
 
-- [🧩 @code-pushup/cli](./packages/cli#readme) - **CLI** for **collecting** audit results and **uploading** report to portal
-- [🧩 @code-pushup/core](./packages/core#readme) - implementation of **core business logic** (useful for custom integrations)
-- [🧩 @code-pushup/models](./packages/models#readme) - **schemas and types** for data models (useful for custom plugins or other integrations)
-- [🧩 @code-pushup/utils](./packages/utils#readme) - various **utilities** (useful for custom plugins or other integrations)
+- [🧩 @code-pushup/cli](./packages/cli#readme) - **CLI** for **collecting** audit results and **uploading** report to portal - [![npm](https://img.shields.io/npm/v/%40code-pushup%2Fcli.svg)](https://www.npmjs.com/package/%40code-pushup%2Fcli)
+- [🧩 @code-pushup/core](./packages/core#readme) - implementation of **core business logic** (useful for custom integrations) - [![npm](https://img.shields.io/npm/v/%40code-pushup%2Fcore.svg)](https://www.npmjs.com/package/%40code-pushup%2Fcore)
+- [🧩 @code-pushup/models](./packages/models#readme) - **schemas and types** for data models (useful for custom plugins or other integrations) - [![npm](https://img.shields.io/npm/v/%40code-pushup%2Fmodels.svg)](https://www.npmjs.com/package/%40code-pushup%2Fmodels)
+- [🧩 @code-pushup/utils](./packages/utils#readme) - various **utilities** (useful for custom plugins or other integrations) - [![npm](https://img.shields.io/npm/v/%40code-pushup%2Futils.svg)](https://www.npmjs.com/package/%40code-pushup%2Futils)
 - plugins:
-  - [📦 @code-pushup/eslint-plugin](./packages/plugin-eslint#readme) - static analysis using **ESLint** rules
+  - [📦 @code-pushup/eslint-plugin](./packages/plugin-eslint#readme) - static analysis using **ESLint** rules - [![npm](https://img.shields.io/npm/v/%40code-pushup%2Feslint-plugin.svg)](https://www.npmjs.com/package/%40code-pushup%2Feslint-plugin)
```

Comment:
Since we the same version for all our packages, I would move the NPM badge to the top of the file (next to our License, CI and Codecov badges).

Reply:
It links to different packages in NPM. I would at least have links somewhere...

Reply:
But the NPM link isn't any more useful than the GitHub link, it's the same README. If you need to get to the NPM link, you can always click the badges you added at the top of each package's README.

Reply:
removed